### DISCUSSION TITLE
Avatto Smart Curtain Switch Compatible?

### DISCUSSION THREAD

**Discussion by MarcoSchicke** [arendst/Tasmota#21720] [Q&A]
Hi I'd like to get a Tasmota Curtain switch where more then one fit below each other. The only one I found works with Tuya and I couldn't find manuals for flashing btw. if its compatible. Can I check somewhere if it would be successful? 
The Device is a AVATTO Smart Curtain Switch SCSDE80-03CMB or the N-CS10. I'm not sure if they only have different names or if they are actually different.

[Link to Amazon for SCSDE80-03CMB](https://www.amazon.de/AVATTO-Rolladenschalter-Unterputz-Rolladensteuerung-Sprachsteuerung/dp/B09YY8MH38/ref=cm_cr_arp_d_pdt_img_top?ie=UTF8&th=1)

[Link to Aliexpress for N-CS10](https://de.aliexpress.com/item/1005005131549516.html?src=google&src=google&albch=shopping&acnt=631-313-3945&slnk=&plac=&mtctp=&albbt=Google_7_shopping&gclsrc=aw.ds&albagn=888888&isSmbAutoCall=false&needSmbHouyi=false&src=google&albch=shopping&acnt=631-313-3945&slnk=&plac=&mtctp=&albbt=Google_7_shopping&gclsrc=aw.ds&albagn=888888&ds_e_adid=&ds_e_matchtype=&ds_e_device=c&ds_e_network=x&ds_e_product_group_id=&ds_e_product_id=de1005005131549516&ds_e_product_merchant_id=106357892&ds_e_product_country=DE&ds_e_product_language=de&ds_e_product_channel=online&ds_e_product_store_id=&ds_url_v=2&albcp=19764771094&albag=&isSmbAutoCall=false&needSmbHouyi=false&gad_source=1&gclid=Cj0KCQjw7ZO0BhDYARIsAFttkCjLgV56nfKHuRrup-WYkQyHhaosHsjsVPD78Ppblw_BzlLWPnf2giAaAqQMEALw_wcB&aff_fcid=750bd28fcfaf43be9fe14d57ddc8872b-1719995243871-05877-UneMJZVf&aff_fsk=UneMJZVf&aff_platform=aaf&sk=UneMJZVf&aff_trace_key=750bd28fcfaf43be9fe14d57ddc8872b-1719995243871-05877-UneMJZVf&terminal_id=42704b90080b43cc9c6ac74c186afc36&afSmartRedirect=n)

Thank you in advance :)

**Comment by raphaelbarreiros** [comment]
Hey @MarcoSchicke! Did you found your answer? I'm also wondering the same
